client/src/app.jsx

```diff
@@ -45,6 +48,7 @@ function initAuthenticated(profile, env) {
   }
 }
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
I find it suspicious that Finn added 0 workarounds for the linter but we added 4 or 5 in this commit. Is the linter really wrong?

Reply:
solved by https://github.com/akvo/akvo-lumen/pull/2244/commits/c3878772db1847ddb21e24cb7035a72ee6b20047